src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -183,6 +186,10 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 	}
 
 	async resolveResources(resourceRequestConfig: TerminalResourceRequestConfig, promptValue: string, cursorPosition: number, provider: string): Promise<ITerminalCompletion[] | undefined> {
+		if (resourceRequestConfig.shouldNormalizePrefix) {
+			// for tests, make sure the right path separator is used
+			promptValue = promptValue.replaceAll(/[\\/]/g, resourceRequestConfig.pathSeparator);
+		}
```

Comment:
We want the flag here to eliminate the rest of the `isWindows` checks so we can control this for the unit tests.

Most of Windows shells I expect to work the same, but a special case we need to consider here is that for git bash we want it to only support `/` and use its special style of absolute paths that end up mapping to real Windows paths:

![image](https://github.com/user-attachments/assets/4ecfa140-577e-4f34-b948-37f388240a87)

We may need to do something similar for `/mnt/` in WSL shells.

Reply:
Oh, ok. Will do.

Reply:
I think we still want this `isWindows` check?

https://github.com/microsoft/vscode/blob/0f33b3dacea552905c6d8a72affc58c709d8e76f/src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts#L254-L257